src/stories/Button.stories.jsx

```diff
@@ -1,40 +1,27 @@
 import React from 'react';
 
-import { Button } from './Button';
+import Button from '../components/Button'
 
-// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
 export default {
-  title: 'Example/Button',
-  component: Button,
-  // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
-  argTypes: {
-    backgroundColor: { control: 'color' },
-  },
+    title: "Button",
+    component: Button,
+
+    argTypes: {
```

Comment:
Just for organization purposes, let's update `argTypes` to categorize the `onClick` property as an Event:

```js
    argTypes: {
        backgroundColor: {control: 'color'},
        onClick: {
          table: {
            category: 'Events',
          },
        },
    },
    ```